library/ec2_remote_facts.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python
+#
+# This is a free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
We definitely don't want to include any GPLv3 source code in this repo, could this file be downloaded instead?